### PR TITLE
Update sandman_nl.ts

### PR DESCRIPTION
**Pull request by RockyTDR**
Update Dutch translations (using my unofficial Transifex project)

### CODE REVIEW

**Comment by isaak654** [comment]
Ok, please consider to add a translation for the multilanguage Plus installer here:
https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Languages.iss

File: SandboxiePlus/SandMan/sandman_nl.ts [hunk 1]
```diff
@@ -2826,7 +2814,7 @@ Als leidende processen gedefinieerd zijn, worden alle andere als achterblijvende
     <message>
         <location filename="Forms/OptionsWindow.ui" line="1597"/>
         <source>Configure which processes can access what resources. Double click on an entry to edit it.
-&apos;Direct&apos; File and Key access only applies to program binaries located outside the sandbox.
+'Direct' File and Key access only applies to program binaries located outside the sandbox.
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_nl.ts:2817]: Did you already test the change here? In sandman_en.ts, the same line looks different:

https://github.com/sandboxie-plus/Sandboxie/blob/f35a246167a6f41661861d43d60a48cf00c6e054/SandboxiePlus/SandMan/sandman_en.ts#L2802

Comment (RockyTDR) [reply SandboxiePlus/SandMan/sandman_nl.ts]: @isaak654 I'm not sure why this happens, but it's probably a bug in Transifex. I'll ask the Transifex helpdesk to look into this!
Luckily it's only one faulty string... 
I edited it manually so the problem is resolved for now.

Thanks for noticing!

**Comment by RockyTDR** [comment]
> 
> 
> Ok, please consider to add a translation for the multilanguage Plus installer (possibly in another pull request):
> https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Languages.iss

Done! https://github.com/sandboxie-plus/Sandboxie/pull/1091

**Pull request merged**